components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS2Parser.java

```diff
@@ -964,40 +1224,6 @@ private String getOAuth2SecuritySchemeKey(Swagger swagger) {
         }
     }
 
-    /**
-     * Gets a list of scopes using the security requirements
-     *
-     * @param oauth2SchemeKey OAuth2 security element key
-     * @param operation       Swagger path operation
-     * @return list of scopes using the security requirements
-     */
-    private List<String> getScopeOfOperations(String oauth2SchemeKey, Operation operation) {
-        List<Map<String, List<String>>> security = operation.getSecurity();
-        if (security != null) {
-            for (Map<String, List<String>> requirement : security) {
-                if (requirement.get(oauth2SchemeKey) != null) {
-                    return requirement.get(oauth2SchemeKey);
-                }
-            }
-        }
-        return getScopeOfOperationsFromExtensions(operation);
-    }
```

Comment:
Is this the same as the above static method? Can if so can we keep this without deleting and remove the static method?